src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntityLivingBase.java

```diff
@@ -933,4 +934,8 @@ public void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators)
         return ProjectileLauncher.launch(checkNotNull(projectileClass, "projectile class"), this, checkNotNull(velocity, "velocity"));
     }
 
+    public void setElytraFlying(boolean value) {
+        setFlag(DataConstants.ELYTRA_FLYING_FLAG, value);
```

Comment:
If we're adding support for the flag, can we have a processor for the change value event?

Reply:
Yeah I'll do so when I get a chance, hopefully this weekend. Will need to look at how you've done it with the others.